Добавил 2 PNG файла со схемами

Andreyshabalinn opened the pull request.

Andreyshabalinn committed:
Добавил 2 PNG файла со схемами

sunzo-digital reviewed (approved):
на будущее - столбцы обычно именуют в snake_case. camelCase тоже допустим, но у тебя какой-то микс)

по таблице космопорта:
1) ранги можно вынести в отдельную таблицу (не критично)

по кинотеатру:
1) вижу номер пользователя в ticket, но я бы вынес пользователей в отдельную таблицу. данные о пользователях - критично важные для любого бизнеса, и если сейчас у нас есть только номер пользователя телефона, мы уже можем делать СМС рассылку. + в дальнейшем таблица может быть расширена
2) думаю в таблицу ticket логично было бы добавить session_id

Andreyshabalinn committed:
Поправил согласно PR-y

Andreyshabalinn committed:
Добавил PNG с проектированием БД KolesaUpgrade

~Пацаны не спят, пацаны бд проектируют~

sunzo-digital reviewed (approved):
Все хорошо, есть пара комментов по неймингу:
1) у тебя опять какая-то странная комбинация из snake_case и camelCase. именуй в одном стиле, например в snake_case
2) таблица уже называется KolesaEmployees, нет необходимости перед каждой колонкой указывать суффикс emp.

sunzo-digital commented:
забыл еще коммент - для связи студента с менторами можно создать отдельную таблицу. таким образом, таблица со студентами будет чище и если в будущем мы захотим отказаться от менторства или добавлять несколько менторов для 1 студента - нам будет легче это сделать

Andreyshabalinn committed:
Поправил замечания

Срезал у своей змеи горб (Скажем нет snake_CamelCase-у)
Добавил таблицу для менторства
Убрал emp_ префикс

sunzo-digital commented:
вот этот коммент относился ко всем похожим случаям, как в таблицах со студентами, лекциями и тасками
 > таблица уже называется KolesaEmployees, нет необходимости перед каждой колонкой указывать суффикс emp.

можешь не исправлять, просто запомни, что нет необходимости в этих суффиксах, когда у тебя название таблицы дает необходимый контекст
